share/registrar_list.toml

```diff
@@ -626,24 +760,51 @@
     
     [ultradns.auth_password]
     type = "password"
+
+[valuedomain]
+    [valuedomain.auth_token]
+    type = "string"
+    redact = true
+
+[vercel]
+    [vercel.auth_token]
+    type = "string"
+    redact = true
     
 [vultr]
     [vultr.auth_token]
     type = "string"
     redact = true
 
-[webgo]
+[wedos]
```

Comment:
fairly sure you have to change the `webgo` references below, otherwise the TOML is likely invalid